salt/states/cmd.py

```diff
@@ -1041,7 +1041,7 @@ def script(name,
             'Replace them with runas. '
             'These arguments will be removed in Salt Oxygen.'
         )
-        if kwargs['user'] is not None and runas is None:
+        if 'user' in kwargs and kwargs['user'] is not None and runas is None:
```

Comment:
I was considering that, but wasn't sure if there was some special case behavior if you provide `False` or an empty string that needs to be preserved. I'd be happy to change it to that if you think it's OK.